examples/dswx_s1_sample_runconfig-v3.0.0-rc.1.0.yaml

```diff
@@ -162,27 +162,27 @@ RunConfig:
             dem_file: /home/dswx_user/input_dir/ancillary_data/dem.tif
 
             # Digital elevation model source description
-            dem_file_description: 'DEM'
+            dem_file_description: 'Copernicus DEM GLO-30 2021 WGS84'
 
             # ESA WorldCover map file
             # (Required for opera_dswx_s1)
             worldcover_file: /home/dswx_user/input_dir/ancillary_data/worldcover.tif
 
             # ESA WorldCover map description
-            worldcover_file_description: 'COPERNICUS WORLDCOVER 10m'
+            worldcover_file_description: 'ESA WorldCover 10m 2020 v1.0'
 
             # Reference water body map (Required)
             # e.g., Pekel's water occurrence or seasonality map
             reference_water_file: /home/dswx_user/input_dir/ancillary_data/reference_water.tif
 
             # Reference water description
-            reference_water_file_description: 'PEKELs water'
+            reference_water_file_description: 'JRC Global Surface Water - collection from 1984 to 2021'
```

Comment:
Just curious why we not longer need 'PEKELs water' if it was significant in an earlier version.

Reply:
'PEKELs water' was a placeholder description taken from the sample runconfig in the interface/beta delivery. The sample provided with the gamma delivery updated all ancillary descriptions, so that has been reflected in our example configs.